boards/common/stm32/include/mp1/cfg_clock_default_208.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2020 Savoir-faire Linux
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_common_stm32
+ * @{
+ *
+ * @file
+ * @brief       Default STM32MP1 clock configuration for 208MHz boards
+ *
+ * @author      Gilles DOFFE <gilles.doffe@savoirfairelinux.com>
+ */
+
+#ifndef MP1_CFG_CLOCK_DEFAULT_208_H
+#define MP1_CFG_CLOCK_DEFAULT_208_H
+
+#include "mp1/cfg_clock_common.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Clock PLL settings (208MHz)
+ * @{
+ */
+/* The following parameters configure a 208MHz system clock with HSE (8MHz or
+   16MHz) or HSI (16MHz) as PLL input clock */
```

Comment:
The comment doesn't reflect the defines below (HSE is 24MHz and HSI, I don't know)